### PR TITLE
Add OIDC authentication docs

### PR DESCRIPTION
**Pull request by helenst**
Accompanying documentation for https://github.com/artefactual/archivematica/pull/1536 and https://github.com/artefactual/archivematica-storage-service/pull/517

Connected to archivematica/Issues#1053

### CODE REVIEW

File: admin-manual/security/security.rst [hunk 1]
```diff
@@ -207,6 +207,39 @@ Shibboleth backend configuration in Storage Service
...
 
+.. _oidc-setup:
+
+OIDC setup
+==========
+
+`OIDC <oidc_>`_ (OpenID Connect) is an identity layer on top of OAuth 2.0, providing
+identity verification, authentication and basic information about the end user. This
+feature allows administrators to deploy both the Archivematica Dashboard and Storage
+Service Django applications with Shibboleth authentication.
```
Comment (alexwlchan) [inline admin-manual/security/security.rst]: Is it Shibboleth auth?

Comment (helenst) [reply admin-manual/security/security.rst]: Copy paste error on my part - that should say OIDC :-)

Comment (sallain) [inline admin-manual/security/security.rst]:
```suggestion
Service Django applications with OIDC authentication.
```

Comment (sallain) [reply admin-manual/security/security.rst]: @alexwlchan Is this the only change you see as needed? I can get one of our devs to review and then we can rebase and merge 🎉

**Comment by alexwlchan** [comment]
@sallain I think that was the only change, yep. Happy for it to be reviewed.

**Review by sallain — APPROVED** [review]
Thanks @alexwlchan - looks great. We have marked archivematica/Issues#1053 for 1.12 as we're in the home stretch for 1.11, so I'll merge this once we've created the 1.12 docs branch. From your perspective, you can mark it as done!

**Comment by alexwlchan** [comment]
Rebased against artefactual/1.11.

**Review by sevein — APPROVED** [review]

**Pull request merged**